exist-core/src/main/java/org/exist/xquery/value/SubSequence.java

```diff
@@ -423,6 +423,36 @@ public void clearContext(final int contextId) throws XPathException {
         sequence.clearContext(contextId);
     }
 
+    @Override
+    public boolean containsReference(final Item item) {
+        try {
+            for (final SequenceIterator it = iterate(); it.hasNext(); ) {
+                final Item i = it.nextItem();
+                if (i == item) {
+                    return true;
+                }
+            }
+            return false;
+        } catch (final XPathException e) {
+            return false;
+        }
+    }
+
+    @Override
+    public boolean contains(final Item item) {
+        try {
+            for (final SequenceIterator it = iterate(); it.hasNext(); ) {
+                final Item i = it.nextItem();
+                if (i.equals(item)) {
+                    return true;
+                }
+            }
+            return false;
+        } catch (final XPathException e) {
+            return false;
```

Comment:
Should we not log the error somewhere here?

Reply:
Okay fixed.